lib/normalize.js

```diff
@@ -80,7 +85,7 @@ module.exports = function normalize(path) {
 			result.push(part);
 		}
 	}
-	if(result.length === 1 && /^[A-Za-z]:$/.test(result))
+	if(result.length === 1 && /^[A-Za-z]:$/.test(result[0]))
```

Comment:
This is not actually a bug fix, but rather a clarity refactor. It's because array to string coercion in JS "just works" for single element arrays:

![image](https://user-images.githubusercontent.com/364532/28029604-988951ca-656e-11e7-9597-cdf9041cfe3c.png)